src/main/java/org/hyperledger/fabric/sdk/BlockInfo.java

```diff
@@ -465,6 +465,17 @@ public int getChaincodeInputArgsCount() {
                 return input.getArgs(index).toByteArray();
             }
 
+            public boolean getChaincodeInputIsInit() {
```

Comment:
Since it's public, could you add a bit of Javadoc to let any consumers know what _isInit_ means. I know the code doesn't set a good example in this area, but it's nice not to add to the technical debt!